src/platform/packages/shared/kbn-es/src/utils/docker.test.ts

```diff
@@ -62,6 +63,17 @@ jest.mock('./wait_for_security_index', () => ({
   waitForSecurityIndex: jest.fn(),
 }));
 
+jest.mock('./docker_uiam', () => {
```

Comment:
__note:__ I test `docker_uiam` unit separately to not over-complicate base `docker` tests.